src/ldp/agent/simple_agent.py

```diff
@@ -40,6 +46,10 @@ class SimpleAgentState(BaseModel):
         default=False,
         description="If True, will hide the content of old ToolRequestMessages.",
     )
+    sliding_window: int = Field(
+        default=0,
+        description="Number of previous trajectory transitions to keep in the state. 0 means all previous transitions.",
```

Comment:
Imo `-1` or `None` are better sentinels here, feel free to disagree with me and ignore this one

Reply:
Agreed, let's do None. 0 is a valid value - gives us a stateless agent.

Reply:
ok lets go with -1 then